README.md

```diff
@@ -1464,7 +1555,9 @@ ENV2_TEST=TEST2
 ENV3_TEST=VALUE OF ENV2 IS: ${ENV2_TEST}
 ```
 
-Env files can also be defined globally in the Makefile.toml via **env_files** attribute as follows:
+Paths to environment files can also be defined globally in the `env_files` key of the `Makefile.toml`, which will be loaded in the order they are defined. All paths are relative to the `Makefile.toml` directory containing them.
+
+> **Note:** `env_files` can only be used on a task level, be aware that relative paths will instead be relative **to the current working directory**
```

Comment:
what is this note? its supported on global and task. why did you write only on task?

Reply:
`only` should've been `also`, thanks for the catch